src/test/java/com/fundation/webservice/model/CriteriaThumbnailVideoTest.java

```diff
@@ -0,0 +1,57 @@
+package com.fundation.webservice.model;
+
+import org.junit.Before;
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+
+public class CriteriaThumbnailVideoTest {
+
+    CriteriaThumbnailVideo criteriaTest = new CriteriaThumbnailVideo();
+    @Before
+    public void setUp(){
+        criteriaTest.setSrcPath(Directories.RSRC_DIR.getDir() + "videoTest.ogv");
+        criteriaTest.setDestPath(Directories.RSRC_DIR.getDir());
+        criteriaTest.setTime("00:01:00");
+        criteriaTest.setName("output");
+        criteriaTest.setExt(".jpg");
+    }
+
+    @Test
+    public void getSrcPath_true() {
+        String expected = "rsrc/videoTest.ogv";
+        assertEquals(expected, criteriaTest.getSrcPath());
+    }
+
+    @Test
+    public void getDestPath_true() {
+        String expected = "rsrc/";
+        assertEquals(expected, criteriaTest.getDestPath());
+    }
+
+    public void getTime_true() {
```

Comment:
Missing @Test  annotation, please insert one.

Reply:
Okay, it's fixed.